auth-web/src/store/modules/business.ts

```diff
@@ -58,7 +58,39 @@ export default class BusinessModule extends VuexModule {
     const currentOrganization: Organization = this.context.rootState.org.currentOrganization
 
     // Create an affiliation between implicit org and requested business
-    await OrgService.createAffiliation(currentOrganization.id, requestBody)
+    return OrgService.createAffiliation(currentOrganization.id, requestBody)
+  }
+
+  @Action({ rawError: true })
+  public async updateBusinessName (businessNumber: string) {
+    let isUpdateFailed = false
+    try {
+      const businessResponse = await BusinessService.searchBusiness(businessNumber)
+      if ((businessResponse?.status === 200) && businessResponse?.data?.business?.legalName) {
+        const updateBusinessResponse = await BusinessService.updateBusinessName({
+          businessIdentifier: businessNumber,
+          name: businessResponse.data.business.legalName
+        })
+        if (updateBusinessResponse?.status === 200) {
+          return updateBusinessResponse
+        } else {
+          isUpdateFailed = true
+        }
+      } else {
+        isUpdateFailed = true
+      }
+    } catch (error) {
+      isUpdateFailed = true
+    }
+    if (isUpdateFailed) {
```

Comment:
I mean its perfectly okay to use the isUpdateFailed variable..its so much readable..but do we have to set this at all places ?